commands/command_track.go

```diff
@@ -181,16 +184,24 @@ func findAttributeFiles() []string {
 		paths = append(paths, repoAttributes)
 	}
 
-	filepath.Walk(config.LocalWorkingDir, func(path string, info os.FileInfo, err error) error {
-		if err != nil {
-			return err
+	fchan, errchan := tools.FastWalkGitRepo(config.LocalWorkingDir)
```

Comment:
Nice. What do you think about pulling this out to a private helper function in the `commands` package? I know the logic is duplicated in a few places across this PR, but I'm not so much thinking about that as I want the `findAttributeFiles` func to read cleanly.